docs/references/protocol/transactions/types/ledgerstatefix.md

```diff
@@ -0,0 +1,68 @@
+---
+seo:
+    description: Repair corruptions to the XRP ledger.
+labels:
+  - Utilities, troubleshooting
```

Comment:
Multiple labels are supposed to be on separate lines (yaml list). You can't comma-separate them here.

Reply:
Done.